recipes-ni/sysconfig-settings/sysconfig-settings_1.0.bb

```diff
@@ -9,6 +9,14 @@ SRC_URI = "file://niselectsystemsettings \
            file://systemsettings/fpga_target.ini \
            file://systemsettings/rt_target.ini \
```

Comment:
This file has a `[consoleOut]` section by default. However due to https://github.com/ni/meta-nilrt/blob/b96ac8fd43f9dd01ed4b14162747ab386a1ab72a/recipes-ni/crio-support-scripts/files/nisetconsoleout#L14, if BootFirmwareConsoleOutEnable is not in the firmware envrionment, the `[consoleOut]` section will be removed. By default, the recover media provisions the target as a cRIO target. And it seems like there is no way to change the firmware environment afterwards, at least with VMware. fw_setenv does not do anything on a VM.